sockeye/training.py

```diff
@@ -92,11 +92,16 @@ def _initialize(self,
         target_length = utils.compute_lengths(target)
         labels = mx.sym.reshape(data=mx.sym.Variable(C.TARGET_LABEL_NAME), shape=(-1,))
 
+        # returns a list of losses
         self.model_loss = loss.get_loss(self.config.config_loss)
 
         data_names = [C.SOURCE_NAME, C.TARGET_NAME]
         label_names = [C.TARGET_LABEL_NAME]
 
+        if self.config.config_length_ratio.link != 'none':
+            lenratio_labels = mx.sym.reshape(mx.sym.Variable(C.LENRATIO_LABEL_NAME), shape=(-1,))
```

Comment:
I actually had the same question about the ```label``` 10 lines above. Why is it there reshaped?
I think I copied it from above, and since it worked, didn't return to it.